server/src/main/java/io/crate/analyze/QueriedSelectRelation.java

```diff
@@ -42,7 +41,7 @@
 public class QueriedSelectRelation implements AnalyzedRelation {
 
     private final List<AnalyzedRelation> from;
-    private final List<JoinPair> joinPairs;
+    private final List<Symbol> joinConditions;
```

Comment:
We still keep the join conditions as part of the `QueriedSelectRelation` to derive outputs and splitpoints easily.